d2l-table-style.html

```diff
@@ -19,7 +19,7 @@
 		<style include="d2l-colors">
 			:host {
 				background-color: transparent;
-				border-collapse: separate;
+				border-collapse: separate !important;
```

Comment:
Any way to have a test that would replicate this so we can know if we accidentally remove this in the future? Or at least a comment here, or both?